<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_Valor">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>14</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Valor:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>561</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Sistema de Lista Linear Duplamente Encadeada</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>181</width>
      <height>131</height>
     </rect>
    </property>
    <property name="midLineWidth">
     <number>-2</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/new/prefix2/icon/foto.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>14</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>inicio -&gt; fim:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Iniciofim">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_fiminicio">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>340</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>14</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Fim -&gt; inicio:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>211</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>imagens/soulindo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>311</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_retirarInicio">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Retirar Inicio</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>imagens/delete_FILL0_wght400_GRAD0_opsz24.png</normaloff>imagens/delete_FILL0_wght400_GRAD0_opsz24.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_inserirInicio">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Inserir Inicio</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>imagens/add_FILL0_wght400_GRAD0_opsz24.png</normaloff>imagens/add_FILL0_wght400_GRAD0_opsz24.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_inserirFInal">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Inserir Final</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>imagens/arrow_back_FILL0_wght400_GRAD0_opsz24.png</normaloff>imagens/arrow_back_FILL0_wght400_GRAD0_opsz24.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_acessarInicio">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Acessar Inicio</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>imagens/left_click_FILL0_wght400_GRAD0_opsz24.png</normaloff>imagens/left_click_FILL0_wght400_GRAD0_opsz24.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_acessarFinal">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Acessar Final</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>imagens/left_click_FILL0_wght400_GRAD0_opsz24.png</normaloff>imagens/left_click_FILL0_wght400_GRAD0_opsz24.png</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_retirarFinal">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Retirar Final </string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>imagens/close_FILL0_wght400_GRAD0_opsz24.png</normaloff>imagens/close_FILL0_wght400_GRAD0_opsz24.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
